pyproject.toml

```diff
@@ -3,13 +3,13 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [project]
-name = "mx-sdk-erdpy-core"
-version = "0.3.1"
+name = "multiversx-sdk-core"
+version = "0.3.2"
 authors = [
   { name="MultiversX" },
 ]
 license = "MIT"
-description = "MultiversX: core components of erdpy."
+description = "MultiversX: core components of the Python SDK."
```

Comment:
Maybe we would also want to write "of the MultiversX Python SDK"?

Maybe: "Core components of the MultiversX Python SDK"?

Reply:
Fixed.